library/svg/metadata.py

```
@@ -0,0 +1,229 @@
+import click
+import json
+import tempfile
+import numpy as np
+import cairosvg
+from io import BytesIO
+from PIL import Image
+from pathlib import Path
+from Pylette import extract_colors
+from xml.etree import ElementTree as ET
+from typing import Tuple, Dict
+from PIL import Image
```

Ruff (0.8.2)

Redefinition of unused `Image` from line 7

Remove definition: `Image`

(F811)